### PR TITLE
Added controllers and routes

### PR DESCRIPTION
**Pull request by zatkiller**
Renamed "Category" schema to "Categories" because sequelize auto-converts all table names to plural and schema names should be plural (Will update Miro Diagram)
Added Category Controllers and Routes
Added CategoryStall Controllers and Routes

Will do documentation once PR pass

### CODE REVIEW

**Comment by ivantjh** [comment]
I updated the `prettier` config, may need to lint again

**Comment by ivantjh** [comment]
I highly doubt the `CategoryStall` controller will be used but we can still leave it there first.

File: src/controllers/categoryController.ts [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+  try {
+    res.status(200).json(req.category);
+  } catch (err) {
+    next(err);
+  }
+}
+
+async function createCategory(req: Request, res: Response, next: NextFunction) {
+  try {
+    const category = await Category.create(req.body);
```
Comment (ivantjh) [inline src/controllers/categoryController.ts]: This should return with the `include` option for consistency across the API calls.

File: src/controllers/categoryStallController.ts [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+  try {
+    res.status(200).json(req.categoryStall);
+  } catch (err) {
+    next(err);
+  }
+}
+
+async function createCategoryStall(req: Request, res: Response, next: NextFunction) {
+  try {
+    const categoryStall = await CategoryStall.create(req.body);
```
Comment (ivantjh) [inline src/controllers/categoryStallController.ts]: Probably need to return with the `include` option too.

**Review by ivantjh — APPROVED** [review]

File: src/controllers/categoryController.ts (hunk 1 above)
Comment (zatkiller) [reply src/controllers/categoryController.ts]: For the other controllers there's no include option in the create except for the Stall Controller

**Pull request merged**